File: Golang/AddSum.go
package main

import "fmt"


func twoSum(nums []int, target int) []int {
    
    result := make([]int, 2)
    for i:=0; i<len(nums);i++ {
        for a:=i+1; a<len(nums);a++ {
            if nums[i]+nums[a]==target{
                result[0]=i;
                result[1]=a;
                return result
            }
        }
    }
    return result
}

func main (){
	nums := []int{2,7,11,15}
	result := twoSum (nums, 9)

	fmt.Println(result)

}

File: Java/AddSum.java

class AddSum {
    public static int[] twoSum(int[] nums, int target) {
        int[] result= new int[2];
        
        for (int i=0; i<nums.length; i++){
            for (int k=i+1; k<nums.length;k++){
                if (nums[i]+nums[k]==target){
                    result[0]=i;
                    result[1]=k;
                    return result;
                }
            }
        }
         return result;
    }   
    
    public static void main(String[] args){
        int[] nums= {2,7,11,15};
        int[] answ= twoSum(nums, 9);
        for (int i : answ){
            System.out.print (i);
        }
        System.out.println();
    }
}